pkg/dependency/interface.go

```diff
@@ -2,13 +2,22 @@ package dependency
 
 import (
 	"context"
+
+	"github.com/OpenFunction/cli/pkg/dependency/inventory"
+)
+
+const (
+	OpenFunctionDir = ".ofn"
+	RecordFileName  = "record.yaml"
```

Comment:
record.yaml is a bit confusing actually. What kind of record?
Maybe we can just call this file `inventory.yaml`

Reply:
renamed it